Cannot write file '/browser-fs-access/index.d.ts' because it would overwrite input file.

@developit: There is a follow-up issue to #97 with microbundle that I'm not entirely sure how to resolve. It seems I can't really tell microbundle where the types are located without causing it to try to do stuff with them:

```bash
rpt2: options error TS5055: Cannot write file '/Users/tsteiner/Documents/javascript/browser-fs-access/index.d.ts' because it would overwrite input file.
  Adding a tsconfig.json file will help organize projects that contain both TypeScript and JavaScript files. Learn more at https://aka.ms/tsconfig.
rpt2: options error TS5055: Cannot write file '/Users/tsteiner/Documents/javascript/browser-fs-access/index.d.ts' because it would overwrite input file.
  Adding a tsconfig.json file will help organize projects that contain both TypeScript and JavaScript files. Learn more at https://aka.ms/tsconfig.
```

--- a/package.json
+++ b/package.json
@@ -22,7 +22,7 @@
   "scripts": {
     "start": "npx http-server -o /demo/",
     "clean": "shx rm -rf ./dist",
-    "build": "npm run clean && microbundle -f modern,cjs --sourcemap false",
+    "build": "npm run clean && microbundle -f modern,cjs --no-sourcemap --no-generateTypes",
     "dev": "microbundle watch",
     "prepare": "npm run lint && npm run fix && npm run build",
     "lint": "npx eslint . --ext .js,.mjs --fix --ignore-pattern dist/",

